gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/QueryResults.java

```diff
@@ -35,8 +35,9 @@
   Class<?> resultClass();
 
   /**
-   * Returns the Cursor for point after the value returned in the last {@link #next} call.
-   * Not currently implemented (depends on v1beta3).
+   * Returns the Cursor for the point after the value returned in the last {@link #next} call. Until
+   * v1beta3 is supported, this field should only be used if you have set a limit and the number of
```

Comment:
I don't think `v1beta3` says much to people. Instead I think we should just say "currently cursorAfter returns null in all cases but the last result" or something like that.